### PR TITLE
docs(one-signal): Add setup documentation for onesignal push provider

### PR DESCRIPTION
**Pull request by shanehandley-lived**
### What change does this PR introduce?

Adds some documentation for the OneSignal push provider.

### Why was this change needed?

Documentation is currently absent, and it required some trial-and-error to set `credentials` correctly.

### CODE REVIEW

File: docs/docs/channels/push/one-signal.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+sidebar_label: One Signal
+sidebar_position: 1
```
Comment (p-fernandez) [inline docs/docs/channels/push/one-signal.md:3]:
```suggestion
sidebar_position: 4
```
Position 1 is already occupied by FCM doc.

File: docs/docs/channels/push/one-signal.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+Before triggering the notification to a subscriber(user) with push as a step in the workflow, make sure you have added the subscriber's `userId` from OneSignal as their device token. This is also referred to as the `Player ID` within the OneSignal web console:
+
+```ts
+import { Novu, PushProviderIdEnum } from '@novu/node';
+
+const novu = new Novu(process.env.NOVU_API_KEY);
+
+await novu.subscribers.setCredentials('subscriberId', PushProviderIdEnum.OneSignal, {
+  deviceTokens: ['token1', 'token2'],
+});
```
Comment (p-fernandez) [inline docs/docs/channels/push/one-signal.md:23]: Could you in the code snippet represent what you are explaining to set the device token, so it is more clear for other users that have less experience with OneSignal and our platform? 🙏🏻

**Review by p-fernandez — COMMENTED** [review]
Thanks for adding this documentation.
🌟

File: docs/docs/channels/push/one-signal.md (hunk 2 above)
Comment (shanehandley-lived) [reply docs/docs/channels/push/one-signal.md]: Updated, cheers 👍

**Review by p-fernandez — APPROVED** [review]
🌟

**Pull request merged**